querybook/webapp/components/ListDataDoc/components/DataDocSchedule.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import cronstrue from 'cronstrue';
+
+export const DataDocSchedule: React.FunctionComponent<{
+    cron: string;
+}> = ({ cron }) => {
+    if (cron) {
+        const friedlyCronSchedule = cronstrue.toString(cron, {
+            verbose: true,
+            use24HourTimeFormat: true,
+        });
+
+        return <div title={friedlyCronSchedule}>{friedlyCronSchedule}</div>;
+    }
+
+    return <div />;
```

Comment:
the preferred syntax is
https://mauriziopireddu.com/clean-code/the-bouncer-pattern
```
if (!cron) {
    return <div />; 
}
return ...
```